site/en/docs/privacy-sandbox/attribution-reporting/index.md

```diff
@@ -34,6 +34,10 @@ If you're unfamiliar with some of these terms, consult the
 
 Keep track of the [API changes](/docs/privacy-sandbox/attribution-reporting-updates/).
 
+{% Aside 'note' %}
+Attribution Reporting is the new name for the Event Conversion Measurement API. [The name was changed](/docs/privacy-sandbox/attribution-reporting-introduction/), as the original event-level scope expanded to cover additional measurement use cases.
```

Comment:
```suggestion
Attribution Reporting was formerly known as the Event Conversion Measurement API. [The name was changed](/docs/privacy-sandbox/attribution-reporting-introduction/) in 2022, as the original event-level scope expanded to cover additional measurement use cases.
```

It's not really "new" anymore, as it's been over a year. However, it's still worth the note, hence the rewrite.